### PR TITLE
updated versions and authors

### PR DESCRIPTION
**Pull request by pfefferle**

### CODE REVIEW

**Comment by pfefferle** [comment]
@1and1/wordpress please review

File: README.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 # Auto Updater #
-**Contributors:** [1and1](https://profiles.wordpress.org/1and1), [markoheijnen](https://profiles.wordpress.org/markoheijnen), [pfefferle](https://profiles.wordpress.org/pfefferle), [gdespoulain](https://profiles.wordpress.org/gdespoulain)  
+**Contributors:** [1and1](https://profiles.wordpress.org/1and1), [ionos](https://profiles.wordpress.org/ionos), [markoheijnen](https://profiles.wordpress.org/markoheijnen), [pfefferle](https://profiles.wordpress.org/pfefferle), [gdespoulain](https://profiles.wordpress.org/gdespoulain)
```
Comment (rykerstribe) [inline README.md:2]: Why still 1and1 in here?

**Review by rykerstribe — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (pfefferle) [reply README.md:2]: it is the old account, that might still be linked from various sources. If we remove the plugin from the account, some might be confused.

File: README.md [hunk 2]
```diff
@@ -48,6 +49,9 @@ When your installation doesn’t allow major updates the settings will not be vi
 
 Project and support maintained on github at [1and1/wp-site-manager](https://github.com/1and1/wp-site-manager).
 
+### 1.1.3 ###
```
Comment (gdespoulain) [inline README.md:52]: Could the comment be a little more self-explaining? ^^ You didn't just increased the version

**Review by rykerstribe — APPROVED** [review]

**Review by kimalumalu — APPROVED** [review]

**Pull request merged**